API_DOCS.md

```diff
@@ -164,8 +164,8 @@ For full instructions check the [`backend README`](./backend/README.md)
 
 **Returns**: An object containing:
   - Success (Boolean) - a success value.
-  - users (List) - List of users matching the query.
```

Comment:
Can you revert these? Everything should be capitalised in the types in the docs; the few that aren't should be fixed

Reply:
Apologies. Overzealous find replace 😉